_posts/2017-11-15-containerised-application-deployment-solutions.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Deployment solutions of a containerised application
+date: 2017-11-15
+---
+
+I have spent part of last week pulling my hair out over container deployments. If you have read my previous posts, you know by now that I have developed a very simple application and packaged into a Docker container. I am currently trying to deploy this application and I never thought deployment would be the challenging part. I looked at container deployments offered by cloud providers because I thought it would be the easiest and cheapest solutions. It probably is the cheapest but not necessarily the easiest!
+
+In this post I am sharing my experience with four cloud providers that offer container deployment service. If you are reading this article, bear in mind that I am no expert in cloud deployments and I am looking at a very particular use case: deployment of a Docker container that needs good availability (it is okay if it goes down once in a while) and does not need to be scalable (one instance running is enough).
+
+#### Search criteria
+
+Obviously, the solution needs to cover my use case, i.e. deployment of a web application packaged into a Docker container. On top of that here are my criterias:
+
+* **Price**
+<br/>The project I am working on is just a pet project and I won't be making any money with it, so the cheaper, the better.
+
+* **Configuration**
+<br/>I am after a solution easy to configure. My intention is not to become a cloud deployment expert and spend ages learning how to use a particular deployment service, so the easier to configure and learn the better.
+
+* **One-click deployment**
+<br/>The deployment itself must be easy and provides an API or a way for me to easily integrate it with my continuous integration pipeline.
```

Comment:
I think 'provide~~s~~' fits better here as the verb is modified by the auxiliary 'must'

Reply:
argh, hate those mistakes.